src/index.html

```diff
@@ -1,13 +1,353 @@
+             grid__item
+             grid__item--tablet-1-3
+             grid__item--desktop-1-2">
+              <div class="about__number-container about__number-container-tablet">
+                <span class="about__number">2015</span>
+                <p class="about__number__subtitle">Foundation year</p>
+              </div>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-5-6
+             grid__item--desktop-3-4">
+              <div class="about__number-container about__number-container-tablet">
+                <span class="about__number">23</span>
+                <p class="about__number__subtitle">Stores</p>
+              </div>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-1-2
+             grid__item--desktop-5-6">
+              <div class="about__number-container">
+                <span class="about__number">155</span>
+                <p class="about__number__subtitle">Companies</p>
+              </div>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-5-6
+             grid__item--desktop-7-8">
+              <div class="about__number-container-modify">
+                <span class="about__number">10k+</span>
+                <p class="about__number__subtitle">Clients</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+      <footer class="page__section footer" id="footer">
+        <div class="page-container">
+          <div class="grid grid--desktop">
+            <div class="
+             grid__item
+             grid__item--tablet-1-2
+             grid__item--desktop-1-2">
+              <a href="#">
+                <img
+                 src="images/footer/footer-logo.svg"
+                 class="footer__logo--img"
+                 alt="logo"
+                >
+              </a>
+              <ul class="footer__list">
+                <li class="footer__item footer__item__title">Menu</li>
+                <li class="footer__item"><a href="#products" class="footer__link">Products</a></li>
+                <li class="footer__item"><a href="#about-us" class="footer__link">About Us</a></li>
+                <li class="footer__item"><a href="#blog" class="footer__link">Blog</a></li>
+                <li class="footer__item"><a href="#terms" class="footer__link">Terms & Privacy</a></li>
+              </ul>
+            </div>
+            <div class="
+             grid__item
+             grid__item--tablet-5-7
+             grid__item--desktop-5-8">
+              <h1 class="footer__title">Contact us</h1>
+              <div class="footer__list-container">
+                <ul class="footer__list">
+                  <li class="footer__item footer__item__title">Call us</li>
+                  <li class="footer__item"><a href="tel:+629-555-0129" class="footer__link">(629) 555-0129</a></li>
```

Comment:
You need to fix format phone only in href
```suggestion
                  <li class="footer__item"><a href="tel:+6295550129" class="footer__link">(629) 555-0129</a></li>
```